### ISSUE TITLE
Can't start Windows based agent in ACI

### ISSUE THREAD

**Issue by chvjak** [JetBrains/teamcity-azure-agent#127]
## Description

When trying to start a Windows based containerized build agent on Azure Container Instances I'm getting: "image  doesn't support specified OS 'Linux'."

## Environment

* TeamCity version: 2020.2.2 (build 85899)
* Azure plugin version: 0.9.5

## Diagnostic logs

[2021-02-17 13:30:47,667]   INFO [atcher-worker-5] - ouds.azure.arm.AzureCloudImage - Creating new virtual machine tc-agent1
[2021-02-17 13:30:47,667]   INFO [-nio-80-exec-10] - .server.impl.CloudEventsLogger - Cloud instance entered 'starting' state, profile 'cp-aci'{id=arm-1, projectId=project1}, AzureCloudInstance{myName='tc-agent1'}
[2021-02-17 13:30:47,667]   INFO [-nio-80-exec-10] - .server.impl.CloudEventsLogger - Cloud instance start succeeded: profile 'cp-aci'{id=arm-1, projectId=project1}, AzureCloudInstance{myName='tc-agent1'}
[2021-02-17 13:31:19,691]   WARN [atcher-worker-5] - ouds.azure.arm.AzureCloudImage - Failed to create deployment tc-agent1 in resource group MC_doa-aks1_doaAKSCluster_westeurope: At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details. The container image 'doaacr1.azurecr.io/tc-agent3:v1' doesn't support specified OS 'Linux' for container group 'tc-agent1'. (ImageOsTypeNotMatchContainerGroup) (BadRequest): jetbrains.buildServer.clouds.CloudException: Failed to create deployment tc-agent1 in resource group MC_doa-aks1_doaAKSCluster_westeurope: At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details. The container image 'doaacr1.azurecr.io/tc-agent3:v1' doesn't support specified OS 'Linux' for container group 'tc-agent1'. (ImageOsTypeNotMatchContainerGroup) (BadRequest) (enable debug to see stacktrace)
[2021-02-17 13:31:19,692]   INFO [atcher-worker-5] - uds.base.AbstractCloudInstance - Changing tc-agent1(1fa918dd) status from Starting to Error

**Comment by chvjak** [comment]
Also, should mention that for Image Type "Container" there is no OS Type input visible in most recent plugin version, but there used to be one in older versions and it worked for me.

**Comment by MSZ-Spyro** [comment]
@chvjak Any updates on this one ?

**Comment by chvjak** [comment]
not really. Abandoned the idea. There are other plugins BTW e.g the ones with  VMSS or AKS as backend. AKS one worked for me.

**Comment by worldspawn** [comment]
So... set the type to Image. Select windows from the drop down, then set the type back to container. Voila, its now set up for windows.
